cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/client/AzureClient.java

```diff
@@ -984,6 +985,12 @@ public DiskEncryptionSetInner createDiskEncryptionSet(String diskEncryptionSetNa
         });
     }
 
+    public boolean checkKeyVaultExistence(String resourceGroupName, String vaultName) {
```

Comment:
I see there are some other `checkXXX()` methods as well already, but `keyVaultExists()` would sound more natural, and imply what the return value expresses.